modules/ossm-validating-smcp.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-validating-smcp_{context}"]
+= Validating the Service Mesh control plane installation
+
+When you create the Service Mesh control plane, the {ProductShortName} Operator uses the parameters that you have specified in the `ServiceMeshControlPlane` YAML file to do the following:
```

Comment:
```suggestion
When you create the Service Mesh control plane, the {ProductShortName} Operator uses the parameters that you have specified in the `ServiceMeshControlPlane` resource file to do the following:
```
I think this reads better since ServiceMeshControlPlane isn't necessarily the literal name of the YAML file, it's just the type of CR?